kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/KieServerConstants.java

```diff
@@ -112,6 +112,8 @@
     public static final String KIE_SERVER_ROUTER = "org.kie.server.router";
     public static final String KIE_SERVER_ROUTER_ATTEMPT_INTERVAL = "org.kie.server.router.connect";
 
+    public static final String KIE_OPTAPLANNER_THREAD_POOL_QUEUE_SIZE = "org.optaplanner.server.ext.thread.pool.queue.size";
```

Comment:
Why the ".ext" in the name? Which other properties do that convention too?

Reply:
This property is used only for OptaplannerKieServerExtension and it's identified by "org.optaplanner.server.ext.XXX".

https://github.com/kiegroup/droolsjbpm-integration/blob/7.23.x/kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/KieServerConstants.java#L31

However, if you suggest any property name, I will follow :)

Reply:
Agreed, this name makes sense. Thanks!